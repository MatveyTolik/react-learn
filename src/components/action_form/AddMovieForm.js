import React from 'react'

export default class EditMovieForm extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <form className={'add-movie-form'}>
        <input type="button" name="close" value="X" className={'close'}/>
        <label className={'form-title'}>ADD MOVIE:</label>
        <label className={'field-title'}>TITLE</label>
        <input type="text"/>
        <label className={'release-date'}>RELEASE DATE</label>
        <input type="date"/>
        <label className={'movie-url'}>MOVIE URL</label>
        <input type="text"/>
        <label className={'genre'}>GENRE</label>
        <select name="genre">
          <option value="Val1">Val 1</option>
          <option value="Val2">Val 2</option>
          <option value="Val2">Val 3</option>
        </select>
        <label className={'overview'}>OVERVIEW</label>
        <input type="text"/>
        <label className={'runtime'}>RUNTIME</label>
        <input type="text"/>
        <input className={'reset'} type="button" name="reset" value="reset"/>
        <input className={'submit'} type="submit" value="submit"/>
      </form>
    );
  }
}
