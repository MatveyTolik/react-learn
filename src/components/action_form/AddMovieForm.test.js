import React from 'react';
import renderer from 'react-test-renderer';
import AddMovieForm from './AddMovieForm';

it('Snapshot to AddMovieForm', () => {
  const tree = renderer.create(<AddMovieForm />).toJSON();
  expect(tree).toMatchSnapshot();
});
