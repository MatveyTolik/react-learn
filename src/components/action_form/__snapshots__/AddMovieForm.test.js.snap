// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot to AddMovieForm 1`] = `
<form
  className="movie-form"
>
  <input
    className="close"
    name="close"
    type="button"
    value="X"
  />
  <label
    className="form-title"
  >
    ADD MOVIE:
  </label>
  <label
    className="field-title"
  >
    TITLE
  </label>
  <input
    name="title"
    onBlur={[Function]}
    onChange={[Function]}
    type="text"
    value=""
  />
  <label
    className="release-date"
  >
    RELEASE DATE
  </label>
  <input
    name="release_date"
    onBlur={[Function]}
    onChange={[Function]}
    type="date"
    value=""
  />
  <label
    className="movie-url"
  >
    MOVIE URL
  </label>
  <input
    name="movie_url"
    onBlur={[Function]}
    onChange={[Function]}
    type="text"
    value=""
  />
  <label
    className="genre"
  >
    GENRE
  </label>
  <select
    name="genre"
  >
    <option
      value="Val1"
    >
      Val 1
    </option>
    <option
      value="Val2"
    >
      Val 2
    </option>
    <option
      value="Val2"
    >
      Val 3
    </option>
  </select>
  <label
    className="overview"
  >
    OVERVIEW
  </label>
  <input
    name="overview"
    onBlur={[Function]}
    onChange={[Function]}
    type="text"
    value=""
  />
  <label
    className="runtime"
  >
    RUNTIME
  </label>
  <input
    name="runtime"
    onBlur={[Function]}
    onChange={[Function]}
    type="text"
    value=""
  />
  <button
    className="reset"
    name="reset"
    type="reset"
  >
    Reset
  </button>
  <button
    className="submit"
    disabled={false}
    onClick={[Function]}
    type="submit"
  >
    Submit
  </button>
</form>
`;
